Scripts/Modules/SerializedPreferencesModule.cs

```diff
@@ -94,8 +95,10 @@ internal string SerializePreferences()
 				}
 
 				if (item != null)
-					preferences.items.Add(item);
+					payloads[serializer.GetType()] = item;
```

Comment:
Echoing @AndrewTHEManeri's comment, too. Not sure why this is necessary. Was this used for debug purposes at one point? I'd say revert back to what it was.